subprojects/docs/src/samples/signing/conditional/build.gradle

```diff
@@ -14,6 +14,13 @@ signing {
 }
 // END SNIPPET conditional-signing
 
+// Alternative to signing.required
+// START SNIPPET only-if
+tasks.withType(Sign)*.onlyIf {
+    isReleaseVersion
+}
```

Comment:
What's the difference between this syntax and:

    tasks.withType(Sign) {
        onlyIf { isReleaseVersion }
    }

? I want to avoid showing too  many ways of doing things in the user manual.

Reply:
The latter is preferred, as it uses less Groovy magic :)

Reply:
I will change it to the latter which is evaluated lazily while the `*` triggers an eager evaluation.